MimeKit/AttachmentCollection.cs

```diff
@@ -86,9 +86,7 @@ public AttachmentCollection () : this (false)
 		/// Indicates the number of attachments in the collection.
 		/// </remarks>
 		/// <value>The number of attachments.</value>
-		public int Count {
-			get { return attachments.Count; }
-		}
+		public int Count => attachments.Count;
```

Comment:
My OCD has so far prevented me from really liking the `=>` style of property declaration, but I'll think about it.

Over the past few years at work, I've been bombarded with this style all over the place (but it's generally inconsistent because "new" code will use this style but older code remains in the previous style, so we have a mish-mash of these styles).

Reply:
I tend to use expression bodies when there's a single line of logic, and use blocks for anything more complicated. But they do look out of place when mixed. I've omitted these changes in #895